src/hb-ot-hmtx-table.hh

```diff
@@ -288,9 +274,9 @@ struct hmtxvmtx
 
     public:
     bool has_font_extents;
-    int ascender;
-    int descender;
-    int line_gap;
+    hb_position_t ascender;
+    hb_position_t descender;
+    hb_position_t line_gap;
```

Comment:
No, these are not hb_position_t.  Keep int.

Reply:
Done